src/components/forms/fields/DateField.tsx

```diff
@@ -0,0 +1,50 @@
+import { useState } from "react"
+import { useFormikContext, FormikValues } from "formik"
+
+import { IFormVariable } from "@/utils/types"
+
+import FieldErrorMessage from "@/components/forms/FieldErrorMessage"
+
+import DatePicker from "react-datepicker"
+import "react-datepicker/dist/react-datepicker.css"
```

Comment:
Are we able to make the React Datepicker look like our custom theme?

Reply:
We can customize using this library https://www.npmjs.com/package/react-datepicker